src/olympia/constants/base.py

```diff
@@ -75,6 +75,16 @@
 VALID_ADDON_STATUSES = (STATUS_NOMINATED, STATUS_APPROVED)
 VALID_FILE_STATUSES = (STATUS_AWAITING_REVIEW, STATUS_APPROVED)
 
+RECOMMENDATION_STATUS_CHOICES = {
```

Comment:
Re-using statuses mmm... Cute, and I appreciate that it helps us avoid death by a thousand constants, but I feel like this will be more confusing than helping. Not strongly opposed to it though

Reply:
I agree, it'll mix- and match different meanings of constants which will be confusing when grepping for a specific constant, see it's usage etc.

Reply:
https://github.com/mozilla/addons-server/pull/11275/commits/9e44673d0f3352b206ac6e188f2ea2c1d1a10dc3 makes the constants redundant now.